models/profiles.yaml

```diff
@@ -69,6 +71,29 @@ var_groups:
           name: days_since_last_seen
           select: "{{macro_datediff('{{user.Var(\"max_timestamp_bw_tracks_pages\")}}')}}"
           default: 0
+        # New boolean feature based on days_since_last_seen
+      - entity_var:
+          name: has_seen_recently
+          select: CASE 
+                    WHEN random() > 0.99 THEN null
+                    WHEN {{user.Var("days_since_last_seen")}} <= 7 
+                    THEN true
+                    ELSE false END
+          description: Whether the user has added a product to seen recently (within the last 7 days).
+        # New categorical feature based on days_since_last_seen
+      - entity_var:
+          name: last_seen_category
+          select: CASE
+                  WHEN random() > 0.99 THEN null
+                  WHEN {{user.Var("days_since_last_seen")}} <= 7 THEN 'recent'
+                  WHEN {{user.Var("days_since_last_seen")}} <= 30 THEN 'moderate'
+                  ELSE 'inactive' END  
+          description: Categorical variable indicating seen activity.
+      # New array type feature based on days_since_last_seen
+      - entity_var:
+          name: last_seen_days_array
+          select: CASE WHEN random() > 0.99 THEN null ELSE TO_ARRAY({{user.Var("days_since_last_seen")}}) END
```

Comment:
This'll just convert the integer column having value for example 14 into new column having value [14]. That's it.

Create Array column from categorical columns having multiple values like ["abc", "xyz", "pqr"]. Use can use any of LISTAGG, ARRAY_AGG or STRING_AGG and see which one works for this usecase.